_data/projects.yml

```diff
@@ -2,7 +2,7 @@
   link: https://coronavirus.ohio.gov/wps/portal/gov/covid-19/
   title: columbus.gov
   description: COVID-19 Resources from the City of Columbus
-  tags: 
+  tags:
```

Comment:
good catch. must not have been working.

Reply:
actually, nvm. i see what happened now. you removed the space.

Reply:
so im not sure if my editor automagically does that or not, because I didn't :P